js/map.js

```diff
@@ -73,49 +98,76 @@ function initialize() {
         var newNode = addMarker(map, results, nodeName, nodeLatLng);
 
         //If new node was created (rather then updated) add it to the marker array
-        if (newNode)
+        if (newNode) 
           nodes.push(newNode);
-      }
-
-      //Draw cone
-      if (results['antennaDirection'] != undefined) {
-        var antennaCone = results['antennaCone'];
-        var antennaDirection = results['antennaDirection'];
-        var antennaDistance = results['antennaDistance'];
-
-        var startArc = antennaDirection - (antennaCone / 2);
-        if (startArc < 0) startArc = startArc + 365;
-        var arcPts = drawArc(nodeLatLng, startArc, startArc + antennaCone, antennaDistance);
-        var piePoly = new google.maps.Polygon({
-          paths: [arcPts],
-          strokeColor: '#0000FF',
-          strokeOpacity: 0.2,
-          strokeWeight: 2,
-          fillColor: '#0000cc',
-          fillOpacity: 0.10,
-          map: map
-        });
-        if (piePoly) { /*Make code climate happy*/ }
-      }
-
-      //Draw links
-      if (results['router'] != undefined) {
-        var routerNode = nodeData[results['router']];
-        if (routerNode != undefined) {
-          var routerLink = [
-            { lat: results['latitude'], lng: results['longitude'] },
-            { lat: routerNode['latitude'], lng: routerNode['longitude'] },
-          ];
-
-          links[results['router'] + '-' + results['name']] = new google.maps.Polyline({
-            path: routerLink,
-            geodesic: true,
-            strokeColor: '#FF0000',
-            strokeOpacity: 1.0,
+        markers[results['name']]=newNode
+
+        //Draw cone
+        if (results['antennaCone'] != undefined) {
+          var antennaCone = results['antennaCone'];
+          var antennaDirection = results['antennaDirection'];
+          var antennaDistance = results['antennaDistance'];
+
+          var startArc = antennaDirection - (antennaCone / 2);
+          if (startArc < 0) startArc = startArc + 365;
```

Comment:
Presumably `startArc` will never be less than `-365`. If it is I expect there would be problems here, but then this line isn't new.